cmd/podman/varlink/io.podman.varlink

```diff
@@ -641,6 +641,13 @@ method StartContainer(name: string) -> (container: string)
 # ~~~
 method StopContainer(name: string, timeout: int) -> (container: string)
 
+# InitContainer initializes the given container. It accepts a container name or
+# ID, and will initialize the container matching that ID if possible, and error
+# if not. Containers can only be initialized when they are in the Created or
+# Exited states. Initialization prepares a container to be started, but does not
```

Comment:
Can you add this description to the actual command --help.  I had no idea what init meant until I read this.
` Initialization prepares a container to be started, but does not start the container.`

Reply:
Can you give me the user an idea why i would EVER want to do this?

Reply:
Added some more context

Reply:
Can you add that to the `--help` as well? Including perhaps a blurb about this being debug-only, not intended for regular users, etc etc? I can easily imagine a new user seeing this option and wasting much time trying to figure out how/when/why to use it. In fact I wonder if it shouldn't be marked hidden, and undocumented? Should this be user-visible at all?

Reply:
I can see non-debug uses, but it's very much for advanced users - it's mostly for people who want the OCI spec created by the container so they can inspect and/or modify it.

Reply:
I like it - brief and descriptive. Could you add that wording to the `--help` and the man page?

Reply:
Done